source/examples/rapids-sagemaker-higgs/rapids-higgs.py

```diff
@@ -50,8 +50,9 @@ def main(args):
     parser.add_argument("--max_features", type=float, default=0.2)
 
     # SageMaker parameters
+    # ref: https://docs.aws.amazon.com/sagemaker/latest/dg/model-train-storage.html
     parser.add_argument("--model_output_dir", type=str, default="/opt/ml/output/")
-    parser.add_argument("--data_dir", type=str, default="/opt/ml/input/data/dataset/")
+    parser.add_argument("--data_dir", type=str, default="/opt/ml/input/data/training/")
```

Comment:
If you don't provide a "channel name" (AWS's language) for the input data, it's by default written to `/opt/ml/input/data/training/`.

Or, more specifically... the dataset you provide is named `training` if you don't otherwise give it a name.

Without this change, training fails with `FileNotFound` errors about not being able to find the input data.